syntaxes/graphql.js.json

```diff
@@ -3,43 +3,40 @@
   "injectionSelector": "L:source -string -comment",
   "patterns": [
     {
-      "name": "taggedTemplates",
```

Comment:
That name was applied to all the captures, but didn't seem to do anything. It's optional, so I removed it.